pyboy/plugins/debug.py

```diff
@@ -233,7 +237,14 @@ def stop(self):
             sdl2.SDL_Quit()
 
     def enabled(self):
-        return self.pyboy_argv.get("debug")
+        if self.pyboy_argv.get("debug"):
+            if not sdl2:
+                logger.error("Failed to import sdl2, needed for debug window")
+                return False
+            else:
+                return True
+        else:
+            return False
```

Comment:
We can shorten this down. For example:
```python
if not sdl2:
    logger.error("Failed to import sdl2, needed for debug window")
    return False
return self.pyboy_argv.get("debug")
```

Reply:
Yeah I wasn't thrilled with it, but this function gets called even if the sdl2/debug windows aren't requested, so we don't want to log errors in that case.